hbase-protocol-shaded/pom.xml

```diff
@@ -34,7 +34,7 @@
     <!--Version of protobuf that hbase uses internally (we shade our pb)
          Must match what is out in hbase-thirdparty include.
     -->
-    <internal.protobuf.version>3.21.1</internal.protobuf.version>
+    <internal.protobuf.version>3.21.7</internal.protobuf.version>
```

Comment:
Ditto

Reply:
Will open a new issue if I find a way to do this.